core/trino-spi/src/main/java/io/trino/spi/eventlistener/TableInfo.java

```diff
@@ -25,15 +25,17 @@
     private final String schema;
     private final String table;
     private final String authorization;
+    private final boolean referencedByView;
```

Comment:
this should be added as last item in constructor params, getters, fields and initialized as last